packages/vitest/src/types/config.ts

```diff
@@ -6,6 +7,8 @@ import type { Arrayable } from './general'
 
 export type BuiltinEnvironment = 'node' | 'jsdom' | 'happy-dom'
 
+export type ApiConfig = Pick<CommonServerOptions, 'port' | 'strictPort' | 'host'>
```

Comment:
@patak-dev we can also include the `path` here, I don't remember where I saw some conversation about changing `/__vitest__/` path, maybe you and Anthony?

Reply:
It cannot be done since it is the `vite.base` for the `UI`, and so we cannot change it